lib/utils/uniqueID.js

```diff
@@ -0,0 +1,9 @@
+var index = 0
+
+// This is of course not using UUIDs, but should be totally sufficient
+// for our use case
+function uniqueID () {
+  return 'faux-dom-' + (index++)
+}
```

Comment:
What I don't like is that this *requires* you to build your element and keep it outside of your render method. It implies that you can not generate your element on the fly on every render (which I how I intended the library to be used). I mean, you can do that but you'll have new ids every time so it'll be slow.

I just don't like the statefulness of it, *personally*. I build everything stateless unless state is required because it leads to cleaner and less buggy code that's easier to read in six months time.